go.mod

```diff
@@ -0,0 +1,15 @@
+module github.com/cockroachdb/cockroach-go/v2
+
+go 1.13
```

Comment:
any reason this is not 1.14?

Reply:
no reason -- that's just what was getting merged from the `version-module` branch, and the file was created a while ago. should i change to 1.14 as part of this PR?

Reply:
don't think it matters too much, either way.